pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/bestpractices/xml/JUnitTestsShouldIncludeAssert.xml

```diff
@@ -529,6 +529,23 @@ public class FooTest extends TestCase {
     public void test2() {
         assertThat("doesn't matter", null, Matchers.nullValue());
     }
+}
+        ]]></code>
+    </test-code>
+
+    <test-code>
+        <description>AssertJ and JUnit5 ?</description>
+        <expected-problems>0</expected-problems>
+        <code><![CDATA[
+import org.assertj.core.api.Assertions;
```

Comment:
Actually I think we should have one test for each of those frameworks. JUnit also has `org.junit.jupiter.api.Assertions`

Reply:
Good point, added a new test case